xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -158,14 +160,27 @@ struct MemoViewerDetailLoadedView: View {
         )
         return .handled
     }
+    
+    private func onViewTransclude(
+        address: Slashlink
+    ) {
+        if address.isOurs {
+            notify(.requestDetail(.editor(MemoEditorDetailDescription(address: address))))
+        } else {
+            notify(.requestDetail(.viewer(MemoViewerDetailDescription(address: address))))
+        }
```

Comment:
Am I overlooking a convenience method we already have for this? `.findAndPushDetail`?